src/Walker.py

```diff
@@ -542,6 +580,18 @@ def visit_Sub(self, node):
         """
         return "__sub__"
     
+    def visit_Subscript(self, node):
+        """
+        @node:ast.ast
+        """ 
+        var = self.visit(node.value) #Item being sliced
+        try:
+            self.visit(node.slice)
+            return #May need to return something else
```

Comment:
I think this should return the subtype of the thing that is being sliced. So if it is a str it would return a str, and if it is a collection then it returns its type with its subtype wrapped in it.